crates/uv-python/src/lib.rs

```diff
@@ -1793,7 +1793,7 @@ mod tests {
         })?;
```

Comment:
Note we don't have test coverage for this because we don't test Python discovery with managed interpreters. I'll create an issue to track infrastructure for this.